### ISSUE TITLE
TypeError: expected a string or other character buffer object

### ISSUE THREAD

**Issue by petabytestorage** [jessevdk/cldoc#133]
```
Processing CMakeCXXCompilerId.cpp
Processing bz1db.cpp
Traceback (most recent call last):
  File "/usr/local/bin/cldoc", line 11, in <module>
    load_entry_point('cldoc==1.11.1', 'console_scripts', 'cldoc')()
  File "/usr/local/lib/python2.7/dist-packages/cldoc-1.11.1-py2.7.egg/cldoc/__init__.py", line 57, in run
    run_generate(rest)
  File "/usr/local/lib/python2.7/dist-packages/cldoc-1.11.1-py2.7.egg/cldoc/__init__.py", line 27, in run_generate
    cmdgenerate.run(args)
  File "/usr/local/lib/python2.7/dist-packages/cldoc-1.11.1-py2.7.egg/cldoc/cmdgenerate.py", line 125, in run
    t.process()
  File "/usr/local/lib/python2.7/dist-packages/cldoc-1.11.1-py2.7.egg/cldoc/tree.py", line 181, in process
    sys.stderr.write(d.format)
TypeError: expected a string or other character buffer object
```

**Comment by jessevdk** [comment]
Right, I already fixed this on `master`, but backported that now to 1.11. Should be fixed in 1.11.2

**Comment by petabytestorage** [comment]
confirmed! thanks again everything is working now. closing down my fork since all bugs are resolved. cheers! :-)
